ddl/util/util.go

```diff
@@ -325,3 +326,68 @@ func WrapKey2String(key []byte) string {
 	}
 	return fmt.Sprintf("0x%x", key)
 }
+
+const mysqlSchemaID = int64(1)
+
+func isSysDB(dbID int64) bool {
+	return dbID == mysqlSchemaID
+}
+
+// HasSysDB checks if it has a system database.
+func HasSysDB(job *model.Job) (bool, error) {
+	// TODO: Handle for the schema ID is 0, like ActionCreatePlacementPolicy.
+	switch job.Type {
```

Comment:
I wonder if it's really necessary to enumerate all the possible job types. Can't we determine this by the schema ID or schema name?